tools/nxfuse/mksmartfsimg.sh

```diff
@@ -19,14 +19,19 @@
 #
 # File : mksmartfsimg.sh
 
-source .config
+NXFUSE_TOOL_PATH=`test -d ${0%/*} && cd ${0%/*}; pwd`
+
+BASE_DIR=$NXFUSE_TOOL_PATH/../..
+
+#source $BASE_DIR/os/.config
+. $BASE_DIR/os/.config
 
 BOARDNAME=$CONFIG_ARCH_BOARD
 POSTFIX=_smartfs.bin
-BINDIR=../build/output/bin
+BINDIR=$BASE_DIR/build/output/bin
 BINNAME=$BOARDNAME$POSTFIX
-CONTENTSDIR=../tools/fs/contents
-NXFUSEDIR=../tools/nxfuse
+CONTENTSDIR=$BASE_DIR/tools/fs/contents-smartfs
```

Comment:
Then, how about renaming contents to contents-romfs to keep the same level of smartfs?
or making romfs and smartfs folders under contents?
```
fs --- contents-romfs
    `-- contents-smartfs

fs --- contents --- romfs
               `-- smartfs
```

Reply:
please let me know your opinion of above.